xbmc/utils/StringUtils.h

```diff
@@ -31,11 +31,12 @@
 #undef FMT_DEPRECATED
 #define FMT_DEPRECATED
 #endif
-#include <fmt/format.h>
-
 #include "XBDateTime.h"
 #include "utils/params_check_macros.h"
 
+#include <fmt/format.h>
+#include <fmt/xchar.h>
```

Comment:
It looks like `xchar.h` is new in v8 so we need a compatibility check like this.
```suggestion
#if FMT_VERSION >= 80000
#include <fmt/xchar.h>
#endif
```

Reply:
Thank you!